test/configuration/vimrcKeyRemappingBuilder.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as assert from 'assert';
+import { IKeyRemapping, IVimrcKeyRemapping } from '../../src/configuration/iconfiguration';
+import { vimrcKeyRemappingBuilder } from '../../src/configuration/vimrcKeyRemappingBuilder';
+
+suite('VimrcKeyRemappingBuilder', () => {
+  test('Build IKeyRemapping objects from .vimrc lines', () => {
```

Comment:
maybe a test case with a multi-line vimrc? and with other options that are ignored?